docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -1,15 +1,190 @@
+    "datastream": {
+      "is_write_index": true
+    }
+  }
+}
+-----------------------
+// CONSOLE
+// TEST[continued]
+
+When creating our index, we have to consider a few important configurations that
+are tie our index and our policy together correctly. We need to make sure that
+our index name matches our index template pattern of "datastream-*", which it does. We
+are using the rollover action in our policy, which requires that our index name ends
+with a number. In our case, we used `00001`. This is important so that Rollover can
+increment this number when naming the new index created from rolling over.
+
+Our index creation request leverages its template to apply our settings, but we
+must also configure out rollover alias: "datastream". To do this, we take advantage of
+<<aliases-write-index,write indices>>. This is a way to define an alias to be used for
+both reading and writing, with only one index being the index that is being written
+to at a time. Rollover swaps the write index to be the new index created from rollover,
+and sets the alias to be read-only for the source index.
+
+
+=== Checking progress
+
+Now that we have an index managed by our policy, how do we tell what is going on?
+Which phase are we in? is something broken? This section will go over a few APIs
+and their responses to help us introspect our indices with respect to {ILM}.
+
+With the help of the <<ilm-explain-lifecycle,Explain API>>, we can know things like which
+phase we're in and when we entered that phase. The API will also provide further
+info if errors occurred, or we are blocked on certain checks within actions.
+
+[source,js]
+--------------------------------------------------
+GET datastream-*/_ilm/explain
+--------------------------------------------------
+// CONSOLE
+// TEST[continued]
+
+The above request will retrieve {ILM} execution information for all our managed indices.
+
+
+[source,js]
+--------------------------------------------------
+{
+  "indices": {
+    "datastream-00001": {
+      "index": "datastream-00001",
+      "managed": true,
+      "policy": "datastream_policy",
+      "lifecycle_date_millis": 1538475653281,
+      "phase": "hot",
+      "phase_time_millis": 1538475653317,
+      "action": "rollover",
+      "action_time_millis": 1538475653317,
+      "step": "attempt-rollover",
+      "step_time_millis": 1538475653317
+      "phase_execution": {
+        "policy": "datastream_policy",
+        "phase_definition": {
+          "min_age": "0ms",
+          "actions": {
+            "rollover": {
+              "max_size": "50GB",
+              "max_age": "30d"
+            }
+          }
+        },
+        "version": 1,
+        "modified_date_in_millis": 1539609701576
+      }
+    }
+  }
+}
+--------------------------------------------------
+// CONSOLE
+// TESTRESPONSE[skip:not possible to get the cluster into this state in a docs test]
+
+There is a lot of information to explain here. For now, let's focus on how
```

Comment:
```suggestion
You can read about the full details of this response in the <<INSERT LINK, explain API docs>>.  For now, let's focus on how
```

Reason: I'd like to try to avoid language that might put the user off here since this is a getting started guide we should make the feature feel as accessible as possible